bundle/manifests/siteconfig-siteconfig-cluster-instance-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
```

Comment:
Why are there two services here?

Reply:
Remnants from older `make bundle` that don't get auto cleaned up.